skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -647,3 +643,19 @@ internal fun Canvas.clipRectBy(rectangle: ClipRectangle, scale: Float) {
         true
     )
 }
+
+// TODO Recheck this method validity in 2 cases - full Window content, and a Panel content
+//  https://youtrack.jetbrains.com/issue/CMP-5447/Window-white-line-on-the-bottom-before-resizing
+/**
+ * Change [value] size to avoid rounding errors, when converting AWT Int coordinates to pixels.
```

Comment:
What does it mean "avoid rounding errors"? What rounding errors? The link to the ticket does help though.

But why 0.4 and 0.6 specifically?

Should we maybe instead use the same function that determines the real content size and compare it to the window/container size? If it's smaller, then increase the unscaled size.

e.g.
```
return if ((value * contentScale).roundToInt() < windowSize)
    value + 1
else
    value
```
(assuming the scaled/real content size is `(value * contentScale).roundToInt()`).

Anyway, this is not a blocker for this PR in any way.

Reply:
Rounding errors when converting Int points (10) to pixels (12) with float scale (1.25)

I will change comment, but I will not change the logic in this PR, as it isn't relevant to it, and I also don't know the whole context of the fix itself, just knowledge of the issue.

Reply:
The link to the issue is in the comment above, this is directly related.

As far as I understand, with this code there was an attempt to increase the size of the component, to avoid white line (but instead overlapped one meaningful content line)